src/content/add-to-app/ios/add-flutter-screen.md

```diff
@@ -660,14 +668,14 @@ in `lib/other_file.dart` instead of `main()` in `lib/main.dart`:
 {% tab "Swift" %}
 
 ```swift
-flutterEngine.run(withEntrypoint: "myOtherEntrypoint", libraryURI: "other_file.dart")
+flutterEngine.run(withEntrypoint: "myOtherEntrypoint", libraryURI: "package:your_package_name/other_file.dart")
```

Comment:
@gaaclarke is this right? The libraryURI should be `package:your_package_name/other_file.dart`?

https://github.com/flutter/engine/pull/30726

Reply:
This sounds like advice that will not apply to everyone.  So at a bare minimum it should be an added note instead of modifying the prevailing guidance.

In order to change the guidance I'd like to see a bug issued.  In order to add the note I'd like to see an example project that demonstrates it's correctness.